### PR TITLE
Issue 25 feature expert popover

### PR DESCRIPTION
**Pull request by DelFlumen**
develop
ExpertPopover
ExpertDataCard

## GitHub Board

**Issue link**

[#25  ]( )

## Code reviewers
- [ ] @github_username

## Summary of issue

- [ ]  Summary of issue
- [ ]  Summary of issue

## Summary of change

- [ ]  Summary of change
- [ ]  Summary of change

### CODE REVIEW

File: src/lib/components/ExpertDataCard.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  expert: IExpert;
+}
+
+const ExpertDataCard: React.FC<IExpertDataCardProps> = (props) => {
+  const { expert } = props;
+
+  let expertFullName = '';
+  if (expert.firstName && expert.secondName) {
+    expertFullName = `${expert.firstName} ${expert.secondName}`;
+  }
```
Comment (SergeOlabin) [inline src/lib/components/ExpertDataCard.tsx:20]:
```suggestion
  const fullName = `${expert.firstName} {expert.secondName}`;
```

File: src/lib/components/ExpertDataCard.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  let directionCyrillic = '';
+  if (expert.direction) {
+    directionCyrillic = directionProperties[expert.direction].cyrillic;
+  }
```
Comment (SergeOlabin) [inline src/lib/components/ExpertDataCard.tsx:24]:
```suggestion
  const directionLabel = directionProperties[expert.direction]?.cyrillic || _FALLBACK_HERE;
```

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: src/lib/components/ExpertDataCard.tsx (hunk 1 above)
Comment (DelFlumen) [reply src/lib/components/ExpertDataCard.tsx]: fixed

File: src/lib/components/ExpertDataCard.tsx (hunk 2 above)
Comment (DelFlumen) [reply src/lib/components/ExpertDataCard.tsx]: fixed

**Review by SergeOlabin — APPROVED** [review]

**Pull request merged**